src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -186,15 +186,24 @@ private static MethodInfo GetMethodInfo(Delegate d)
         /// <summary>A cleanup handle to the Process created for the remote invocation.</summary>
         public sealed class RemoteInvokeHandle : IDisposable
         {
-            public RemoteInvokeHandle(Process process, RemoteInvokeOptions options, string assemblyName = null, string className = null, string methodName = null)
+            private const int DefaultExitCode = 0;
+            public RemoteInvokeHandle(Process process, RemoteInvokeOptions options, int exitCode, string assemblyName = null, string className = null, string methodName = null)
```

Comment:
you have introduced this exitCode but you didn't store it inside the object.

I suggest you remove the changes in the constructor and just keep the ExitCode property only. and in the property setter, you may check if we are not in UAP and throw at that time. then in RemoteExecutorTestBase.uap.cs after creating RemoteInvokeHandle object, set the ExitCode property.

Reply:
Or you can do:

```cs
return new RemoteInvokeHandle(...)
{
    ExitCode = res;
};